raco/language/myrialang.py

```diff
@@ -806,6 +838,9 @@ def __repr__(self):
     def num_tuples(self):
         return self._num_tuples
 
+    def partitioning(self):
```

Comment:
I may be mistaken, but I thought `MyriaQueryScan` could push any SQL to Postgres. In general, this could include Apply's and whatever else and so is not trivial like `MyriaScan`

I would propose that if it is important at some point to make this not so conservative that we consider getting the partitioning info from the SQL query fragment's plan
(https://github.com/uwescience/raco/blob/master/raco/language/myrialang.py#L1470).